pkg/assetfetcher/http.go

```diff
@@ -183,7 +183,13 @@ func (c *NexposeAssetFetcher) newNexposeSiteAssetsRequest(siteID string, page in
 func (a Asset) hasBeenScanned() bool {
 	for _, evt := range a.History {
```

Comment:
Question: aren't we walking the asset history in `AssetPayloadToAssetEvent()` and here?